CSLabs.Api/Proxmox/ProxmoxApi.cs

```diff
@@ -81,6 +82,23 @@ public async Task<NodeStatus> GetNodeStatus(HypervisorNode node = null)
 
         }
         
+        public async Task<NodeStatus> GetClusterStatus(HypervisorNode node = null)
+        {
+            if (node == null)
+            {
+                node = HypervisorNode;
+            }
+            await LoginIfNotLoggedIn();
+            var response = await PerformRequest(() => this.client.Cluster.Status.GetStatus());
+            var data = response.Response.data;
+            var nodeStatus = new NodeStatus
```

Comment:
Use a new model called `ClusterStatus` instead since `NodeStatus` is the status for the node, not the cluster.